### PR TITLE
Updated code with safe checks

### PR DESCRIPTION
**Pull request by artem-v**
Checks like: empty `tag`, ` containerLogPath` an `containerLogHostBaseDir`. Added ability to specify long timeout when grep log.

### CODE REVIEW

File: src/main/java/io/scalecube/docker/utils/Containers.java [hunk 1]
```diff
@@ -167,6 +174,17 @@ public static void restartContainer(Container container) {
...
+  }
+
+  /**
+   * @param container container reference.
+   * @param sinceMillis UTC millis to filter logs; it's mandatory for outputing log-entries since only that timestamp.
+   * @param longTimeout timeout to wait for operation completion.
+   * @param tokens tokens to match; will be quoted and joined with together '.*'.
+   * @return millis corresponding to matched log line.
+   */
+  public static long grepLog(Container container, long sinceMillis, int longTimeout, String... tokens) {
```
Comment (snripa) [inline src/main/java/io/scalecube/docker/utils/Containers.java]: `int longTimeout -> int timeout`

File: src/main/java/io/scalecube/docker/utils/Containers.java [hunk 2]
```diff
@@ -175,7 +193,7 @@ public static long grepLog(Container container, long sinceMillis, String... toke
 
       boolean completion = dockerClient.logContainerCmd(container.getContainerName())
           .withStdOut(true).withStdErr(true).withTimestamps(true).withSince(unixTime).withFollowStream(true)
-          .exec(callback).awaitCompletion(LONG_TIMEOUT, TimeUnit.SECONDS);
+          .exec(callback).awaitCompletion(longTimeout, TimeUnit.SECONDS);
```
Comment (snripa) [inline src/main/java/io/scalecube/docker/utils/Containers.java]: `int longTimeout -> int timeout`

**Comment by antonkharenko** [comment]
@artem-v Lets maybe get rid from all this timeout constants LONG/SHORT and pass timeout also to execCommand

File: src/main/java/io/scalecube/docker/utils/Containers.java (hunk 1 above)
Comment (antonkharenko) [inline src/main/java/io/scalecube/docker/utils/Containers.java]: also specify either it is seconds or millis

**Pull request merged**